app/views/notifications/_emergency_banner.html.erb

```diff
@@ -1,9 +1,9 @@
-<section id="emergency-banner-notification" class="<%= banner.campaign_class %>">
+<section id="emergency-banner-notification" class="<%= banner.campaign_class %>" role="banner">
   <div>
-    <p><%= banner.heading %><br>
-    <%= banner.short_description %></p>
+    <h1><%= banner.heading %></h1>
+    <p><%= banner.short_description %></p>
     <% if banner.link %>
-      <a href="<%= banner.link %>" class="more-information">More information</a>
+      <a href="<%= banner.link %>">More information</a>
```

Comment:
In order to [enable custom link text](https://trello.com/c/ZPkk7hnk/51-add-new-field-for-link-text), some more work needs to be done. As the blitz is over, they're out of scope for this PR.